synapse/python_dependencies.py

```diff
@@ -95,7 +95,7 @@
     "oidc": ["authlib>=0.14.0"],
     "systemd": ["systemd-python>=231"],
     "url_preview": ["lxml>=3.5.0"],
-    "test": ["mock>=2.0", "parameterized"],
+    "test": ["mock>=2.0", "parameterized>=0.7.0"],
```

Comment:
Actually, I seem to be experiencing an issue with `0.7.0`:

![image](https://user-images.githubusercontent.com/1342360/84499789-2c400680-acab-11ea-905b-2fe8d10a2191.png)

I do not experience this with `0.7.1`. I guess we make use of `setUp` in `MediaRepoTests`: https://github.com/wolever/parameterized/blob/ff5a1dac3a704777100380229dce4ea149da8d41/CHANGELOG.txt#L22

Reply:
I don't see any issues with 0.7.0.

Reply:
Requiring 0.7.1 is undesirable because Fedora 32 has 0.7.0.

Reply:
Odd, I have an issue locally but CI running with `0.7.0` seems to be fine...

I'm just going to chalk this up to something to do with my python environments I guess.